MAINTAINERS.md

```diff
@@ -89,31 +89,42 @@ A simple majority vote is required to authorize a SPIRE release at a specific co
 This section summarizes the steps necessary to execute a SPIRE release. Unless explicitly stated, the below steps must be executed in order.
 
 The following steps must be completed one week prior to release:
-* Ensure all changes intended to be included in the release are fully merged
-* Identify a specific commit as the release candidate
-* Raise an issue "Release SPIRE X.Y.Z", and include both the release candidate commit hash and the proposed changelog updates
+* Ensure all changes intended to be included in the release are fully merged.
+* Identify a specific commit as the release candidate.
+* Create a draft pull request with the updates to the CHANGELOG. This allows to have publicly available the changes that will be included in the upcoming release with some anticipation, providing the oportunity to receive feedback. The release date can be set as "TBD".
+  * The CHANGELOG distinguishes two types of changes: "What's New" and "What's Changed". As a general rule, the entries should be focused on exposing user-facing changes, considering that the main consumers of the CHANGELOG are the end users of SPIRE.
```

Comment:
what do you think about moving towards a model like this? https://keepachangelog.com/en/1.0.0/#how

```
Types of changes
 - Added: for new features.
 - Changed: for changes in existing functionality.
 - Deprecated: for soon-to-be removed features.
 - Removed: for now removed features.
 - Fixed: for any bug fixes.
 - Security: in case of vulnerabilities.
```

this way, we're clear to users about when things get deprecated and removed, so they know what needs to be updated

Reply:
I really like that!